org.x.adriconf.json

```diff
@@ -0,0 +1,183 @@
+{
+    "app-id": "org.x.adriconf",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "18.08",
+    "sdk": "org.freedesktop.Sdk",
+    "command": "adriconf",
+    "finish-args": [
+         /*In future we need wayland also!*/
+         "--socket=x11",
+         "--share=ipc",
+         "--device=dri",
+         "--talk-name=org.freedesktop.secrets",
+         "--system-talk-name=org.freedesktop.GeoClue2"
+    ],
+
```

Comment:
the blank lines aren't our usual flathub formatting practice but that's not a hard blocker for me.